lib/l10n/app_it.arb

```diff
@@ -0,0 +1,600 @@
+  "collectionRenameFailureFeedback": "{count, plural, =1{Impossibile rinominare 1 elemento} other{Impossibile rinominare {count} elementi}}",
+  "collectionEditFailureFeedback": "{count, plural, =1{Impossibile modificare 1 elemento} other{Impossibile modificare {count} elementi}}",
+  "collectionExportFailureFeedback": "{count, plural, =1{Impossibile esportare 1 elemento} other{Impossibile esportare {count} elementi}}",
+  "collectionCopySuccessFeedback": "{count, plural, =1{Copiato 1 elemento} other{Copiati {count} elementi}}",
+  "collectionMoveSuccessFeedback": "{count, plural, =1{Spostato 1 elemento} other{Spostati {count} elementi}}",
+  "collectionRenameSuccessFeedback": "{count, plural, =1{Rinominato 1 elemento} other{Rinominati {count} elementi}}",
+  "collectionEditSuccessFeedback": "{count, plural, =1{Modificato 1 elemento} other{Modificati {count} elementi}}",
+
+  "collectionEmptyFavourites": "Nessun preferito",
+  "collectionEmptyVideos": "Nessun video",
+  "collectionEmptyImages": "Nessuna immagine",
+
+  "collectionSelectSectionTooltip": "Seleziona sezione",
+  "collectionDeselectSectionTooltip": "Deseleziona sezione",
+
+  "drawerCollectionAll": "Tutte le collezioni",
+  "drawerCollectionFavourites": "Preferiti",
+  "drawerCollectionImages": "Immagini",
+  "drawerCollectionVideos": "Video",
+  "drawerCollectionAnimated": "Animato",
+  "drawerCollectionMotionPhotos": "Foto in movimento",
+  "drawerCollectionPanoramas": "Panorami",
+  "drawerCollectionRaws": "Foto raw",
+  "drawerCollectionSphericalVideos": "Video a 360°",
+
+  "chipSortDate": "Per data",
+  "chipSortName": "Per nome",
+  "chipSortCount": "Per numero di elementi",
+
+  "albumGroupTier": "Per livello",
+  "albumGroupVolume": "Per volume di archiviazione",
+  "albumGroupNone": "Non raggruppare",
+
+  "albumPickPageTitleCopy": "Copia su album",
+  "albumPickPageTitleExport": "Esportazione su album",
+  "albumPickPageTitleMove": "Sposta su Album",
+  "albumPickPageTitlePick": "Scegli l'album",
+
+  "albumCamera": "Camera",
+  "albumDownload": "Download",
+  "albumScreenshots": "Screenshots",
+  "albumScreenRecordings": "Registrazioni schermo",
+  "albumVideoCaptures": "Video Captures",
+
+  "albumPageTitle": "Album",
+  "albumEmpty": "Nessun album",
+  "createAlbumTooltip": "Crea album",
+  "createAlbumButtonLabel": "CREA",
+  "newFilterBanner": "nuovo",
+
+  "countryPageTitle": "Paesi",
+  "countryEmpty": "Nessun paese",
+
+  "tagPageTitle": "Etichette",
+  "tagEmpty": "Nessuna etichetta",
+
+  "binPageTitle": "Cestino",
+
+  "searchCollectionFieldHint": "Cerca raccolta",
+  "searchSectionRecent": "Recenti",
+  "searchSectionAlbums": "Album",
+  "searchSectionCountries": "Paesi",
+  "searchSectionPlaces": "Luoghi",
+  "searchSectionTags": "Etichette",
+  "searchSectionRating": "Valutazioni",
+
+  "settingsPageTitle": "Impostazioni",
+  "settingsSystemDefault": "Sistema",
+  "settingsDefault": "Default",
```

Comment:
I'm surprised there's no word for this in Italian. Are you sure you want to keep the English?

Reply:
There sure is, it just sounded better to me.. But just to be complete we can put `Predefinite/Predefiniti` (here I need to know the gender of the word where it is used though, assuming it's used for plural)